src/agent/driver-test-runner.js

```diff
@@ -91,27 +91,37 @@ export class DriverTestRunner {
   }
 
   /**
-   * @param {"reading" | "interactive"} mode
+   * Used for v2 aria-at tests, "reading" and "interactive" map to various settings based on AT.
```

Comment:
The references to "reading" and "interactive" (or maybe "interaction") sound more like the v1 format. Is this description accurate for this method?

Reply:
dbebce3 removes this and lets the doc on the "settings" param do the job of what values it expects.